### PR TITLE
DS lowercase

### PR DESCRIPTION
**Pull request by charlene1804**

### CODE REVIEW

File: doubt-removers/src/content.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default {
       },
       {
         title: 'Publish npm packages',
-        description: `Design Systems are compiled, packaged and versioned on demand. 
+        description: `design systems are compiled, packaged and versioned on demand.
```
Comment (gqio) [inline doubt-removers/src/content.js:39]: I think at the beginning of a phrase it should be uppercase

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@divriots/lcd",
-  "version": "0.2.17",
+  "version": "0.2.18",
```
Comment (gqio) [inline package.json:3]: please rebase on master

File: doubt-removers/src/content.js (hunk 1 above)
Comment (charlene1804) [reply doubt-removers/src/content.js]: Fixed 🙏🏻

**Comment by gqio** [comment]
Still conflict on the package.json. I think you will need to rebase this branch on main

**Review by gqio — APPROVED** [review]

**Pull request merged**